dev-libs/orcus/orcus0.20-0.20.2.recipe

```diff
@@ -126,18 +132,25 @@ PATCH()
 BUILD()
 {
 	autoreconf -fi
+	export CXXFLAGS="-std=c++17 -O2"
```

Comment:
This is missing `-g` to get useful debug infos. Or you can remove the debuginfo package.